docs/frontend_architecture/single_page_apps.rst

```diff
@@ -83,16 +83,22 @@ For more information on using `bundle_id` and connecting it to the relevant Java
 This will create a navigation component which will be registered to appear in the navigation side bar.
 
 
-Content renderers
------------------
+Content viewers
```

Comment:
I know this is what we moved towards but i am confused about why the renderers > viewers change is in this commit as I thought it has been merged already